packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -1,165 +1,306 @@
+      ...createLightModeVariantVariables('primary'),
+    },
+    neutral: {
+      ...colors.grey,
+      ...createLightModeVariantVariables('neutral'),
+    },
+    danger: {
+      ...colors.red,
+      ...createLightModeVariantVariables('danger'),
+    },
+    info: {
+      ...colors.blue,
+      ...createLightModeVariantVariables('info'),
+    },
+    success: {
+      ...colors.green,
+      ...createLightModeVariantVariables('success'),
+    },
+    warning: {
+      ...colors.yellow,
+      ...createLightModeVariantVariables('warning'),
+    },
+    text: {
+      primary: 'var(--joy-palette-neutral-800)',
+      secondary: 'var(--joy-palette-neutral-600)',
+      tertiary: 'var(--joy-palette-neutral-500)',
+    },
+    background: {
+      default: 'var(--joy-palette-neutral-50)',
+      level1: '#fff',
+      level2: 'var(--joy-palette-neutral-100)',
+      level3: 'var(--joy-palette-neutral-200)',
+    },
+    focusVisible: 'var(--joy-palette-primary-200)',
+  },
+  elevationRing: '0 0 #000',
+};
+
+export const darkColorSystem: Pick<BaseJoyTokens, 'palette' | 'elevationRing'> = {
   palette: {
-    brand: colors.purple,
-    neutral: colors.grey,
+    primary: {
+      ...colors.purple,
+      ...createDarkModeVariantVariables('primary'),
+    },
+    neutral: {
+      ...colors.grey,
+      ...createDarkModeVariantVariables('neutral'),
+    },
+    danger: {
+      ...colors.red,
+      ...createDarkModeVariantVariables('danger'),
+    },
+    info: {
+      ...colors.blue,
+      ...createDarkModeVariantVariables('info'),
+    },
+    success: {
+      ...colors.green,
+      ...createDarkModeVariantVariables('success'),
+    },
+    warning: {
+      ...colors.yellow,
+      ...createDarkModeVariantVariables('warning'),
+    },
     text: {
-      heading: 'var(--joy-palette-neutral-900)',
-      headingIntro: 'var(--joy-palette-brand-300)',
-      content: 'var(--joy-palette-neutral-600)',
-      detail: 'var(--joy-palette-neutral-500)',
-      overline: 'var(--joy-palette-neutral-500)',
+      primary: 'var(--joy-palette-neutral-100)',
+      secondary: 'var(--joy-palette-neutral-300)',
+      tertiary: 'var(--joy-palette-neutral-400)',
     },
-    bgNeutral: {
-      transparency: 'var(--joy-palette-neutral-100)',
-      plain: 'var(--joy-palette-neutral-100)',
+    background: {
+      default: 'var(--joy-palette-neutral-800)',
+      level1: 'var(--joy-palette-neutral-900)',
+      level2: 'var(--joy-palette-neutral-700)',
+      level3: 'var(--joy-palette-neutral-600)',
     },
+    focusVisible: 'var(--joy-palette-primary-400)',
   },
+  elevationRing: '0 0 #000',
+};
+
+/**
+ * Base Joy design tokens
+ * Any value with `var(--joy-*)` can be used. 'joy-' will be replaced by the application prefix if provided.
+ */
+const joyDesignTokens: BaseJoyTokens = {
+  ...lightColorSystem,
   borderRadius: {
-    md: '4px',
+    default: '28px',
+    xs: '4px',
+    sm: '8px',
+    md: '12px',
+    lg: '20px',
+    xl: '32px',
```

Comment:
```suggestion
    default: '16px',
    xs: '4px',
    sm: '8px',
    md: '12px',
    lg: '18px',
    xl: '24px',
```

This can change as we iterate but for now, I think I'd go with these values